openpype/hosts/maya/plugins/publish/collect_loaded_plugin.py

```diff
@@ -0,0 +1,17 @@
+import pyblish.api
+from maya import cmds
+
+
+class CollectLoadedPlugin(pyblish.api.ContextPlugin):
```

Comment:
Considering this needs to be fully optional, I reckon it would be better to keep the pluginInfo query inside of the validator itself. That data is not useful anywhere else and it will speed up the collection when it's turned off.

Reply:
Totally agree